test/serializer/trivial/GlobalVariable.js

```diff
@@ -1,4 +1,4 @@
 // does not contain:void 0
 var x = 1;
-x = 2;
```

Comment:
This is really something that test262 should cover. It may well do that already, but we need to update our version of it. Meanwhile it doesn't hurt to just sneak it into an existing test case. Most of those need basic things to work and break when they don't. This is just one more such thing.

Reply:
This test case is in a directory called "trivial". This change makes it very much not trivial. I don't think we should overload existing targeted test cases by sneaking in other complex stuff.